index.js

```diff
@@ -0,0 +1,144 @@
+const header = document.createElement('header');
+
+const headerh1 = document.createElement('h1');
+headerh1.setAttribute('id', 'logo');
+headerh1.innerText = 'Techno Mobile';
+
+const button = document.createElement('button');
```

Comment:
- [ ] Your button should only be visible on mobile resolution. 

It will be confusing for the users if you have both menu and button for opening the menu. You should hide your hamburger button on desktop resolution.